lib/charms/layer/zookeeper.py

```diff
@@ -139,3 +172,17 @@ def update_zoo_cfg(self, zkid=getid(local_unit()), ip=unit_private_ip(), remove=
                 contents.append(key + value + "\n")
             with open(zookeeper_cfg, 'w', encoding='utf-8') as f:
                 f.writelines(contents)
+
+    def run_bg(self, user, output_log, command, *args):
+        """
+        Run a command as the given user in the background.
```

Comment:
Man, this seems familiar.  ;)  We should really refactor this up in to `jujubigdata.utils`